src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -643,6 +643,23 @@ internal static partial class DbConnectionStringDefaults
         internal const string TransactionBinding = "Implicit Unbind";
         internal const int ConnectRetryCount = 1;
         internal const int ConnectRetryInterval = 10;
+
+        // Retry Logic
+        internal const string RetryStrategy = "None";
+        internal const int RetryCount = 3;
+
+        internal const int RetryInterval = 10;
```

Comment:
This seems like the time interval. Suffix the variables with unit of time.

Reply:
Applicable to all internal members.

Reply:
Sounds good!